### PR TITLE
Bump eventing to v1.5

### PR DESCRIPTION
**Pull request by pierDipi**
As per title.

### CODE REVIEW

**Review by matzew — APPROVED** [review]
/lgtm
/approve

/hold
(Just to be safe :trollface: )

**Comment by pierDipi** [comment]
The mesh error is unrelated but
/test 4.11-upstream-e2e-mesh-aws-ocp-411

File: test/lib.bash [hunk 1]
```diff
@@ -199,19 +199,19 @@ function run_rolling_upgrade_tests {
...
-{{- else if eq . "heartbeats"          }}{{.}}:v$eventing_image_version
-{{- else if eq . "performance"         }}{{.}}:v$eventing_image_version
-{{- else if eq . "print"               }}{{.}}:v$eventing_image_version
-{{- else if eq . "recordevents"        }}{{.}}:v$eventing_image_version
-{{- else if eq . "request-sender"      }}{{.}}:v$eventing_image_version
-{{- else if eq . "wathola-fetcher"     }}{{.}}:v$eventing_image_version
-{{- else if eq . "wathola-forwarder"   }}{{.}}:v$eventing_image_version
-{{- else if eq . "wathola-receiver"    }}{{.}}:v$eventing_image_version
-{{- else if eq . "wathola-sender"      }}{{.}}:v$eventing_image_version
+{{- else if eq . "event-flaker"        }}knative-eventing-test-{{.}}:knative-v$eventing_image_version
```
Comment (matzew) [inline test/lib.bash]: so, this was broken?

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -883,12 +883,6 @@ spec:
     - name: "KAFKA_IMAGE_kafka-controller-post-install__post-install"
       image: "registry.ci.openshift.org/openshift/knative-v1.5:knative-eventing-kafka-broker-post-install"
     - name: "KAFKA_IMAGE_knative-kafka-storage-version-migrator__migrate"
-<<<<<<< HEAD
```
Comment (matzew) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml]: :see_no_evil:

File: hack/lib/serverless.bash [hunk 3]
```diff
@@ -347,7 +347,7 @@ function teardown_serverless {
     oc delete knativekafka.operator.serverless.openshift.io knative-kafka -n "${EVENTING_NAMESPACE}"
   fi
   logger.info 'Ensure no knative eventing or knative kafka pods running'
-  timeout 600 "[[ \$(oc get pods -n ${EVENTING_NAMESPACE} --field-selector=status.phase!=Succeeded -o jsonpath='{.items}') != '[]' ]]"
+  timeout 700 "[[ \$(oc get pods -n ${EVENTING_NAMESPACE} --field-selector=status.phase!=Succeeded -o jsonpath='{.items}') != '[]' ]]"
```
Comment (matzew) [inline hack/lib/serverless.bash:350]: let's still break into different jobs, ok?

**Review by matzew — APPROVED** [review]
/lgtm
/approve

File: hack/lib/serverless.bash (hunk 3 above)
Comment (pierDipi) [reply hack/lib/serverless.bash:350]: That's for a different task/subject https://github.com/openshift-knative/serverless-operator/pull/1674 I guess, right?

File: test/lib.bash (hunk 1 above)
Comment (pierDipi) [reply test/lib.bash]: No, this one https://github.com/openshift-knative/serverless-operator/pull/1711/commits/776811291ce4461b0995e8aacaaa42871908f4dc

File: hack/lib/serverless.bash (hunk 3 above)
Comment (pierDipi) [reply hack/lib/serverless.bash:350]: but yes

**Pull request merged**